modules/dedicated-storage-expanding-filesystem-pvc.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * storage/expanding-persistent-volume.adoc
+
+[id="expanding-pvc-filesystem_{context}"]
+= Expanding {product-name} Persistent Volume Claims (PVCs)
+
+Expanding PVCs based on volume types that need file system resizing,
+such as AWS EBS, is a two-step process.
+This process involves expanding volume objects in the cloud provider, and
+then expanding the file system on the actual node. These steps occur automatically
+after the PVC object is edited, and may require a pod restart to take effect.
+
+Expanding the file system on the node only happens when a new pod is started
+with the volume.
+
+.Prerequisites
+
+* The controlling StorageClass must have `allowVolumeExpansion` set
+to `true`. This is the default configuration in {product-name}.
+
++
+[IMPORTANT]
+====
+Decreasing the size of an AWS EBS volume is not supported. However, you
+can create a smaller volume and then migrate your data to it using a
+tool such as `oc rsync`. After modifying a volume, you must wait at least six hours before
+making additional modifications to the same volume.
+====
+
+.Procedure
+
+. Edit the PVC and request a new size by editing
+`spec.resources.requests.storage`. For example, the following expands
+the `ebs` PVC to 8 Gi.
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: ebs
+spec:
+  storageClass: "gp2"
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi <1>
+----
+<1> Updating `spec.resources.requests.storage` to a larger amount will expand
+the PVC.
++
+You can do this with a simple `oc patch` command:
++
+----
+$ oc patch pvc ebs -p '{"spec": {"resources": {"requests": {"storage": "8Gi"}}}}'
+----
+
+. Once the cloud provider object has finished resizing, the PVC is set to
```

Comment:
I think that all of the required information is here, but I'd reorder it.

I'd start with the instruction to run the command, like "Review the status of the PVC:" Then the command and output. Then explain that it's set to  `FileSystemResizePending` after the cloud provider object size adjusted.

Then add a separate step, with the command, to delete the Pod, maybe like "If the output of the previous command included a message to restart the Pod, delete the mounting Pod that it specified:"